aries-site/src/data/wcag/components.json

```diff
@@ -48,5 +48,419 @@
+      },
+      {
+        "rule": "1.4.12",
+        "label": "Text Spacing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "1.4.13",
+        "label": "Content on Hover or Focus",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.1.1",
+        "label": "Keyboard",
+        "status": "conditional",
+        "notes": "Dependent on the card's content and interactivity. If the entire card is a link, use an <a> element. Test that the card and any interactive elements within it can be reached and activated using the keyboard alone."
+      },
+      {
+        "rule": "2.1.2",
+        "label": "No Keyboard Trap",
+        "status": "conditional",
+        "notes": "Requires implementor to test depending on the card's content."
+      },
+      {
+        "rule": "2.1.3",
+        "label": "Keyboard (No Exception)",
+        "status": "conditional"
+      },
+      {
+        "rule": "2.1.4",
+        "label": "Character Key Shortcuts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.1",
+        "label": "Timing Adjustable",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.2",
+        "label": "Pause, Stop, Hide",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.3",
+        "label": "No Timing",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.4",
+        "label": "Interruptions",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.5",
+        "label": "Re-authenticating",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.2.6",
+        "label": "Timeouts",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.1",
+        "label": "Three Flashes or Below Threshold",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.2",
+        "label": "Three Flashes",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.3.3",
+        "label": "Animation from Interactions",
+        "status": "not-applicable",
+        "notes": "I don't think hover elevation is considered here. Check with team."
+      },
+      {
+        "rule": "2.4.1",
+        "label": "Bypass Blocks",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.4.2",
+        "label": "Page Titled",
+        "status": "not-applicable"
+      },
+      {
+        "rule": "2.4.3",
+        "label": "Focus Order",
+        "status": "conditional",
+        "notes": "Depending on context, a proper role of `listitem` or wrapping in an <li> may be needed."
+      },
+      {
+        "rule": "2.4.4",
+        "label": "Link Purpose (In Context)",
+        "status": "conditional",
+        "notes": "Related cards sharing a common call to action such as 'Learn More' may need to be differentiated by using `aria-label` to provide context as to answer 'learn more about what?'."
```

Comment:
Good note, feel like this is something we could be better at demonstrating in our card examples